src/dashboard/src/components/administration/views.py

```diff
@@ -386,9 +386,22 @@ def _usage_get_directory_used_bytes(path):
     except OSError:
         logger.exception("No such directory: %s", path)
         return 0
-    except subprocess.CalledProcessError:
-        logger.exception("Unable to determine usage of %s.", path)
-        return 0
+    except subprocess.CalledProcessError as err:
+        # CalledProcessErrors are typically the result of du returning
+        # 1 because there were some directories in the path for which
+        # the archivematica user doesn't have permissions. In this case
+        # du still prints an output to stdout so we try to catch it
+        # from CalledProcessError.output.
+        byte_count = 0
+        try:
+            byte_count = int(err.output.split("\t")[0])
+            logger.warning(
+                "Non-zero exit code while determining usage of %s. Some directories may be missing from total.",
+                path,
+            )
```

Comment:
Function parameters can take multiline strings without extra symbols, e.g.:

```suggestion
            logger.warning(
                "Non-zero exit code while determining usage of %s. "
                "Some directories may be missing from total.",
                path,
            )
```